README.md

```diff
@@ -56,15 +52,14 @@ into which the project was cloned and run `stack build`.
 The [`stack` system](https://docs.haskellstack.org/en/stable/README/) is
 required before `cardano-sl` can be built.
 
-Running the nodes
------------------
-
-After the project is built, run it:
-
-- open *new terminal window*, run `tmux` in it (`sudo apt-get install tmux` or else according to your distro)
-- open *new terminal window*, run `export WALLET_TEST=1; ./scripts/launch.sh` in it
+For developer convenience, also note that in the `scripts` and `util-scripts` folders
+there are a variety of utility scripts to build various components of the system,
+such as the general `util-scripts/build.sh`, or the `util-scripts/clean.sh` to remove
+byproducts of compilation.
 
-After a while, the first terminal windows should be populated with 4 tabs, 3 out of which will have a node running in them.
+Note also that as of March 2017, it is not possible to use the `cardano-sl:with-web` and
```

Comment:
Is it worth mentioning in README at main page? Seems to me like everybody who reads this README and only this README should be happy with running simple commands like `./util-scripts/build.sh` and don't expect problems. People shouldn't even know about such flags. Or this is crucial for building on windows?